### PR TITLE
Add vuid 06697, Validate RenderPassMultiview view mask bits

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -12919,6 +12919,14 @@ bool CoreChecks::PreCallValidateCreateRenderPass(VkDevice device, const VkRender
         for (uint32_t i = 0; i < multiview_info->subpassCount; ++i) {
             all_zero &= multiview_info->pViewMasks[i] == 0;
             all_not_zero &= !(multiview_info->pViewMasks[i] == 0);
+            if (MostSignificantBit(multiview_info->pViewMasks[i]) >=
+                static_cast<int32_t>(phys_dev_props_core11.maxMultiviewViewCount)) {
+                skip |= LogError(device, "VUID-VkRenderPassMultiviewCreateInfo-pViewMasks-06697",
+                                 "vkCreateRenderPass(): Most significant bit in "
+                                 "VkRenderPassMultiviewCreateInfo->pViewMask[%" PRIu32 "] (%" PRIu32
+                                 ") must be less maxMultiviewViewCount(%" PRIu32 ").",
+                                 i, multiview_info->pViewMasks[i], phys_dev_props_core11.maxMultiviewViewCount);
```
Comment (Tony-LunarG) [inline layers/core_validation.cpp:12928]: must be less _than_ maxMultiviewViewCount

File: tests/vklayertests_descriptor_renderpass_framebuffer.cpp [hunk 2]
```diff
@@ -10274,6 +10274,15 @@ TEST_F(VkLayerTest, RenderPassMultiViewCreateInvalidViewMasks) {
 
     ASSERT_NO_FATAL_FAILURE(InitState());
 
+    auto render_pass_mutliview_props = LvlInitStruct<VkPhysicalDeviceMultiviewProperties>();
```
Comment (Tony-LunarG) [inline tests/vklayertests_descriptor_renderpass_framebuffer.cpp:10277]: mutliview should be multiview

Comment (Tony-LunarG) [inline tests/vklayertests_descriptor_renderpass_framebuffer.cpp:10277]: mutliview should be multiview

File: tests/vklayertests_descriptor_renderpass_framebuffer.cpp [hunk 3]
```diff
@@ -12181,6 +12190,14 @@ TEST_F(VkLayerTest, RenderPassMultiViewCreateInvalidViewOffsets) {
...
```
Comment (Tony-LunarG) [inline tests/vklayertests_descriptor_renderpass_framebuffer.cpp:12193]: mutliview should be multiview

Comment (Tony-LunarG) [inline tests/vklayertests_descriptor_renderpass_framebuffer.cpp:12193]: mutliview should be multiview

**Review by Tony-LunarG — COMMENTED** [review]
Just a couple of nits

**Review by Tony-LunarG — APPROVED** [review]
LGTM

**Pull request merged**